dev/io.openliberty.mcp.internal/resources/io/openliberty/mcp/internal/CWMCM.nlsprops

```diff
@@ -0,0 +1,153 @@
+###############################################################################
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+###############################################################################
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWMCM
+#COMPONENTNAMEFOR CWMCM MCP Server
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#ISMESSAGEFILE true
+
+# Used for server logging
+# do not translate MCP
+# {0} = The qualified name of the tool
+CWMCM0001E.blank.arguments=CWMCM0001E: The {0} MCP tool method has one or more arguments with blank names
+CWMCM0001E.blank.arguments.explanation=MCP Tool arguments were provided but they had missing names. Each argument to a tool method must have a name.
+CWMCM0001E.blank.arguments.useraction=Provide non-empty values for all required arguments.
+
+# Used for server logging
+# do not translate MCP
+# {0} = The  qualified name of the tool
+# {1} = corresponding argument name
+CWMCM0002E.duplicate.arguments=CWMCM0002E: The {0} MCP tool method has more than one argument named {1}
+CWMCM0002E.duplicate.arguments.explanation=One or more arguments have the same name. Each argument of an MCP tool method must have a different name.
+CWMCM0002E.duplicate.arguments.useraction=Give each tool argument a different name.
+
+# Used for server logging
+# do not translate MCP
+# {0} = The qualified name of the tool
+CWMCM0003E.missing.tool.argument.name=CWMCM0003E:  The {0} MCP tool method has one or more arguments without a name specified
+CWMCM0003E.missing.tool.argument.name.explanation=One or more Tool argument name(s) was not provided for MCP Tools.
+CWMCM0003E.missing.tool.argument.name.useraction=Either add a name to the @ToolArg annotation, or to add the -parameters compiler option to use the parameter name.
+
+# Used for server logging
+# do not translate MCP
+# {0} = The list of qualified names of the tool
+CWMCM0004E.duplicate.tools=CWMCM0004E: There are multiple MCP tool methods named {0}. The methods are {1}.
+CWMCM0004E.duplicate.tools.explanation=One or more MCP Tool names occured multiples times.The server expects one name per tool for uniqueness.
+CWMCM0004E.duplicate.tools.useraction=Only provide a unique name for each MCP Tool.
+
+# Used for server logging
+# do not translate MCP
+CWMCM0005E.validation.error=CWMCM0005E: There are one or more MCP validation errors. Check logs for more details.
```

Comment:
Change to "Logs have one or more MCP validation errors. Check the logs for more details." You could start with "The logs have".

Removes the "There are" and adds "the" before "logs."

Reply:
I'm not sure this is right.

As we start up we're going to validate all of the MCP resources that the user has included and will log out any validation errors.

If there are validation errors, this message is then output in the log as the reason that the application did not start.

"Logs have one or more MCP validation errors" sounds like it's the logs that are at fault, whereas it's the user's application that is at fault but we want them to look back in this log file to fix the validation errors.